project/src/main/world/Overworld.tscn

```diff
@@ -26,21 +22,10 @@
 [ext_resource path="res://src/main/world/OverworldExit.tscn" type="PackedScene" id=24]
 [ext_resource path="res://src/main/world/overworld-bg.gd" type="Script" id=25]
 [ext_resource path="res://src/main/world/environment/MarshTree.tscn" type="PackedScene" id=26]
-[ext_resource path="res://src/main/world/environment/MarshHouse8.tscn" type="PackedScene" id=27]
 [ext_resource path="res://src/main/world/environment/MarshHouse9.tscn" type="PackedScene" id=28]
-[ext_resource path="res://src/main/world/environment/MarshHouse6.tscn" type="PackedScene" id=29]
-[ext_resource path="res://src/main/world/environment/MarshHouse10.tscn" type="PackedScene" id=30]
-[ext_resource path="res://src/main/world/environment/MarshHouse7.tscn" type="PackedScene" id=31]
-[ext_resource path="res://src/main/world/environment/MarshHouse11.tscn" type="PackedScene" id=32]
-[ext_resource path="res://src/main/world/environment/MarshHouse1.tscn" type="PackedScene" id=33]
-[ext_resource path="res://src/main/world/environment/MarshHouse12.tscn" type="PackedScene" id=34]
-[ext_resource path="res://src/main/world/environment/MarshHouse4.tscn" type="PackedScene" id=35]
-[ext_resource path="res://src/main/world/environment/MarshHouse5.tscn" type="PackedScene" id=36]
-[ext_resource path="res://src/main/world/environment/MarshHouse13.tscn" type="PackedScene" id=37]
```

Comment:
Delete these unused house scenes and images.

Reply:
Deleted all `MarshHouseX.tscn` scene files and `marsh-house-x.png` image files. Only MarshHouse9 remains.